akka-stream-testkit/src/main/scala/akka/stream/testkit/TestGraphStage.scala

```diff
@@ -40,22 +49,38 @@ private[testkit] class TestSinkStage[T, M](
         inHandler.onPush()
```

Comment:
if we don't have a reason for doing it differently I think there can be some value in doing it in the same way for all of them